File: abc162/a/a.ts
import * as fs from "fs";

function luck(n: string): boolean {
  if (n.includes('7')) {
    return true;
  }
  return false;
}

function main() {
  const n = fs.readFileSync("/dev/stdin", "utf-8").trim();

  if (luck(n)) {
    console.log("Yes");
  }
  else {
    console.log("No");
  }
}

main();

File: abc162/a/a.rs
use std::io::Read;

fn luck(n: &str) -> bool {
    if n.contains('7') {
        return true;
    }
    return false;
}

fn main(){
    let mut buf = String::new();
    std::io::stdin().read_to_string(&mut buf).unwrap();
    let n = buf.split_whitespace().next().unwrap();

    if luck(n) {
        println!("Yes");
    }
    else {
        println!("No");
    }
}
